console-src/src/types/oAuthClient.tsx

```diff
@@ -1,3 +1,5 @@
+import * as Schema from "@kontist/client/dist/graphql/schema";
```

Comment:
This works for now but shouldn't we expose these types in a more straightforward fashion in our SDK?

Reply:
Makes sense. Done in 931242f. Should now work with
`import { Schema } from "@kontist/client";`

Reply:
I'll update it once we have this included in a release 🙂I'll merge the PR like this and will open another one when appropriate